contrib/test/ci/build/kubernetes.yml

```diff
@@ -82,6 +82,15 @@
       ./hack/local-up-cluster.sh
     mode: "u=rwx,g=rwx,o=x"
 
+- name: Set CGROUP_CONFIG for fedora
+  become: yes
+  lineinfile:
+    dest: /usr/local/bin/createcluster.sh
```

Comment:
Why can't we put this inside the createcluster.sh script?

Reply:
I don't think we're hitting this issue for other variants, so tried the safer approach.